GTG/backends/__init__.py

```diff
@@ -67,10 +67,12 @@ def __init__(self):
                 # Something is wrong with this backend, skipping
                 log.warning("Backend %s could not be loaded: %r",
                             module_name, exception)
+                inactive_modules.append(module_name)
```

Comment:
```python-traceback
Traceback (most recent call last):
  File "/app/lib/python3.9/site-packages/GTG/backends/__init__.py", line 65, in __init__
    __import__(extended_module_name)
  File "/app/lib/python3.9/site-packages/GTG/backends/backend_caldav.py", line 30, in <module>
    import caldav
ModuleNotFoundError: No module named 'caldav'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/app/lib/python3.9/site-packages/GTG/gtk/application.py", line 124, in do_startup
    for backend_dic in BackendFactory().get_saved_backends_list():
  File "/app/lib/python3.9/site-packages/GTG/backends/__init__.py", line 70, in __init__
    inactive_modules.append(module_name)
AttributeError: 'set' object has no attribute 'append'
```

You need to use [`.add()`](https://docs.python.org/3.8/library/stdtypes.html#frozenset.add) since it is a set, not an list.